site/content/completions/_index.md

```diff
@@ -174,8 +174,8 @@ cmd := &cobra.Command{
 	Use:   "status RELEASE_NAME",
 	Short: "Display the status of the named release",
 	Long:  status_long,
-	RunE: func(cmd *cobra.Command, args []string) {
-		RunGet(args[0])
+	Run: func(cmd *cobra.Command, args []string) {
+		RunStatus(args[0])
```

Comment:
Not sure why this is changed but it shouldn't matter either way as it is an example only so I am fine with that.

Reply:
That was just a cleanup. The example is about the `helm status` command but I had left the function call to `RunGet` which was a copy/paste from the `kubectl get` example.